starter-code/src/main.js

```diff
@@ -40,11 +42,58 @@ $(document).ready(function(){
   });
 
   // Add all the div's to the HTML
-  document.getElementById('memory_board').innerHTML = html;
+  $('#memory_board').html(html);
   // Bind the click event of each element to a function
-$('.back').on('click', function () {
-   
-});
-});
+  var cardsCounter = 0;
+  var card1 = 0;
```

Comment:
More semantically correct would be to initiate `card1` and `card2` as `null` or `{}` instead of `0`, since they are never going to be numbers, and `null` commonly represents "nothing"